src/store/modules/blocks/state.js

```diff
@@ -3,6 +3,7 @@
  */
 export default {
   height: 0,
+  generation: {},
   block: {},
-  blocks: []
```

Comment:
`blocks` is still used in other places:
https://github.com/aeternity/aepp-blockchain-explorer/blob/499c71eb3cb6023b8045627e9da8f681c2b84e2e/src/store/modules/blocks/getters.js#L13